kolibri/plugins/coach/assets/src/app.js

```diff
@@ -57,6 +57,13 @@ class CoachToolsModule extends KolibriModule {
           },
         },
         {
+          name: PageNames.COACH_EXERCISE_RENDER_PAGE,
+          path: '/exercise-render',
+          handler: (toRoute, fromRoute) => {
+            actions.showCoachExerciseRenderPage(store);
+          },
+        },
+        {
```

Comment:
Note in the mockups, the Exercise Detail View is essentially a modal that gets loaded over a place where the exercise itself is displayed for an individual learner:

https://projects.invisionapp.com/share/7D8ZDY4EN#/screens/220191559